bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/TextLayout.java

```diff
@@ -2789,7 +2791,7 @@ public int getTextDirection () {
  */
 public int getWidth () {
 	checkLayout();
-	return DPIUtil.autoScaleDown(getDevice(), getWidthInPixels());
+	return getWidthInPixels();
```

Comment:
same as above, if getWidthInPixels does not return pixels, we should rename it

Reply:
Yeah I think there are 2 methods now. Which do the same thing. I'll think something.